throw_error(message):
  display "__uncaught_error"
  __main__() # this will wipe out the call stack!
